### PR TITLE
fix readme and uikit dist dependency

### PR DESCRIPTION
**Pull request by ameli90**

### CODE REVIEW

File: build/webpack.base.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ var projectRoot = path.resolve(__dirname, '../')
 
 module.exports = {
   externals: {
-    uikit: 'UIkit'
+    uikit: 'uikit'
```
Comment (miljan-aleksic) [inline build/webpack.base.js]: Uikit is globally registered under `UIkit` space name. This change is wrong, the others seems ok.

File: build/webpack.dist.js [hunk 2]
```diff
@@ -44,7 +44,6 @@ module.exports = [
       filename: '[name].js',
       chunkFilename: '[id].js'
     },
-    devtool: '#source-map',
```
Comment (miljan-aleksic) [inline build/webpack.dist.js]: Why are you suggesting removing it?

Comment (ameli90) [inline build/webpack.dist.js]: I thought devtool is using to debug and no need to enhance debug in production. But source map supports in production, reverted

File: build/webpack.base.js (hunk 1 above)
Comment (ameli90) [inline build/webpack.base.js]: added to dist config

**Pull request merged**